### PR TITLE
{compiler}[GCCcore/8.3.0] Clang v10.0.0

### PR DESCRIPTION
**Pull request by Falkor**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Falkor** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-057 - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.7.4
See https://gist.github.com/79f82a7394d8783da59ac49f35ae17aa for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node309.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/e7dac78f125422203d53cd99dfde62d4 for a full test report.

File: easybuild/easyconfigs/c/Clang/Clang-10.0.0-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    'clang-%(version)s.src.tar.xz',
+    'compiler-rt-%(version)s.src.tar.xz',
+    'polly-%(version)s.src.tar.xz',
+    'openmp-%(version)s.src.tar.xz',
+    # Also include the LLVM linker
+    'lld-%(version)s.src.tar.xz',
+    'libcxx-%(version)s.src.tar.xz',
+    'libcxxabi-%(version)s.src.tar.xz',
+]
+# patches = ['libcxx-%(version)s-ppc64le.patch']
```
Comment (lexming) [inline easybuild/easyconfigs/c/Clang/Clang-10.0.0-GCCcore-8.3.0.eb:36]: May I ask why is this patch commented out? is it no longer necessary or does it need an update?

Comment (Falkor) [reply easybuild/easyconfigs/c/Clang/Clang-10.0.0-GCCcore-8.3.0.eb:36]: I could not test if it is still required on PPC arch nor if it is stil applicable, thus I leave it out. If you have a way to test it?

Comment (lexming) [reply easybuild/easyconfigs/c/Clang/Clang-10.0.0-GCCcore-8.3.0.eb:36]: @Falkor I checked that patch and it still applies to Clang v10. I made a PR to your branch to help you update this easyconfig. Please check https://github.com/ULHPC/easybuild-easyconfigs/pull/1

Comment (Falkor) [reply easybuild/easyconfigs/c/Clang/Clang-10.0.0-GCCcore-8.3.0.eb]: Thanks @lexming -- it's merged and I added the checksum

**Comment by Falkor** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-019 - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.7.4
See https://gist.github.com/d34d782c28c483e071ec79d840eca8e5 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node150.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/a186b8d1041024eabdde7ca911e40054 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @Falkor !

**Pull request merged**